schemas/com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Common Context Schema for a media player event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "currentTime": {
+      "type": "number",
```

Comment:
Could it be floating point?

Reply:
Yeah, it's a double in the spec we've based this off: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime

> A double-precision floating-point value indicating the current playback time in seconds.